javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/AgentInstaller.java

```diff
@@ -238,9 +245,7 @@ private static void runAfterAgentListeners(
     } else {
       for (AgentListener agentListener : agentListeners) {
         agentListener.afterAgent(config);
-        if (autoConfiguredSdk != null) {
-          agentListener.afterAgent(autoConfiguredSdk);
-        }
+        agentListener.afterAgent(config, autoConfiguredSdk);
```

Comment:
The `afterAgent()` listener method is also executed from the `DelayedAfterAgentCallback` (under some specific conditions), can you pass the `autoConfiguredSdk` there too?